packages/@dcos/sorting/__tests__/compare-as-string-test.ts

```diff
@@ -0,0 +1,30 @@
+import { compareAsString, ComparisonResult } from "../";
+
+const idLense = (x: any) => x;
+
+describe("compareAsString", () => {
+  it("returns 0 if any on the values is null", () => {
```

Comment:
Same issue with strings I guess. 

Why not null being "" here?

Reply:
> Why not null being "" here?

If you mean `"null"` I would say that is a perfectly valid string

Reply:
Nope, I mean the same problem for the numbers. If you say null is the same as any value then sort does not work as expected `also for the other values on the list`. 

I think whatever solves the other comment, solves that one by analogy.

Reply:
@juliangieseke and I had some discussions around this and after this discussion I think we should just not support sorting by null at all. Let's make sure the lens always returns the right type and we should be good to go.

Reply:
So if I understood you correctly, the lens should look at null and give it a meaning inside that type. Which I decide. Could be 0 for Numbers and "" for Strings If I so choose. Is that what you mean?

Reply:
Also, will it be any runtime guarantees?